src/Components/AdvancedSearch/AdvancedSearch.tsx

```diff
@@ -51,6 +53,10 @@ const AdvancedSearchModal: React.FC<IAdvancedSearchProps> = (props) => {
     const updateColumns = (properties: Set<string>) => {
         additionalProperties.current = properties;
     };
+    const stackTokens: IStackTokens = {
+        childrenGap: '20px',
```

Comment:
nit: children gap can just be a number too. That's what we do elsewhere in the app. `childrenGap: 20`